File: code/statistics/python/stat/muestreo/remuestreo-media.py
import pandas as pd
import numpy as np
import matplotlib.pylab as plt

WEB_PAGE_DATA_CSV = '/home/data/Documentos/practical-statistics-for-data-scientists-master/data/web_page_data.csv'

session_times = pd.read_csv(WEB_PAGE_DATA_CSV)
session_times.Time = 100 * session_times.Time

mean_a = session_times[session_times.Page == 'Page A'].Time.mean()
mean_b = session_times[session_times.Page == 'Page B'].Time.mean()
print(mean_b - mean_a)                           

File: ciencia-de-datos/code/statistics/r/stastics/muestreo/permutacion/remuestreo-media.R
library(ggplot2)

session_times <- read.csv('/home/data/Documentos/practical-statistics-for-data-scientists-master/data/web_page_data.csv')
head(session_times)
session_times[,2] <- session_times[,2] * 100

mean_a <- mean(session_times[session_times['Page'] == 'Page A', 'Time'])
mean_b <- mean(session_times[session_times['Page'] == 'Page B', 'Time'])
mean_b - mean_a

